<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiszki @ 2020</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>12</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartOver">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>12</y>
      <width>126</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelShowTerm">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>100</y>
      <width>481</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>211</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>189</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelTerm">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>139</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Hasło&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAddNewTerm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dodaj nowe hasło</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDefinition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Odpowiedź&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditInputTerm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditInputDefinition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddNewTerm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSummary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Podsumowanie</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>38</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="UserAnswer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Twoja odpowiedź: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUserAnswer">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="CorrectMessage">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Poprawna odpowiedź:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCorrectMessage">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="UsersGrade">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Twoja ocena:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUsersGrade">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dalej</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Wpisz odpowiedź</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>461</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditInputAnswer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonShowAnswer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pokaż odpowiedź</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOpenDatabase"/>
    <addaction name="actionAddNewDatabase"/>
    <addaction name="actionSaveDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdycja">
    <property name="title">
     <string>Edycja</string>
    </property>
    <addaction name="actionDeleteTerm"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuEdycja"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolBarOpenDatabase"/>
   <addaction name="toolBarSaveDatabase"/>
   <addaction name="toolBarAddNewDatabase"/>
   <addaction name="toolBarDeleteTerm"/>
   <addaction name="toolBarSaveSession"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Otwórz istniejącą bazę</string>
   </property>
  </action>
  <action name="actionAddNewDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj nową bazę</string>
   </property>
  </action>
  <action name="actionDeleteTerm">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/trash.png</normaloff>:/images/trash.png</iconset>
   </property>
   <property name="text">
    <string>Usuń hasło</string>
   </property>
  </action>
  <action name="toolBarDeleteTerm">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/trash.png</normaloff>:/images/trash.png</iconset>
   </property>
   <property name="text">
    <string>Usuń hasło</string>
   </property>
  </action>
  <action name="toolBarAddNewDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj nową bazę</string>
   </property>
  </action>
  <action name="toolBarOpenDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Otwórz istniejącą bazę</string>
   </property>
  </action>
  <action name="actionSaveDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz bazę</string>
   </property>
  </action>
  <action name="toolBarSaveDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz bazę</string>
   </property>
  </action>
  <action name="actionSaveSession">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/session.png</normaloff>:/images/session.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz sesję</string>
   </property>
  </action>
  <action name="toolBarSaveSession">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/session.png</normaloff>:/images/session.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz sesję</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
